docker/Dockerfile

```diff
@@ -1,7 +1,9 @@
-FROM node:10-jessie-slim as react
-RUN mkdir /rasa-nlu-data
+FROM baseapp as react
```

Comment:
Ao usar o comando `docker-compose up --build`, recebo este erro

> Step 1/7 : FROM baseapp as react
ERROR: Service 'rasa-trainer' failed to build: pull access denied for baseapp, repository does not exist or may require 'docker login'

Reply:
Faltou especificar no README a execução do comando ./builbase.sh

Reply:
Resolvido :)